src/chrome/Runtime.js

```diff
@@ -2,11 +2,13 @@
 
```

Comment:
Ditto.

Reply:
This defines types of the chrome extension `chrome.runtime` api.